turbinia/processors/partitions.py

```diff
@@ -25,11 +25,12 @@
 log = logging.getLogger('turbinia')
 
 
-def Enumerate(evidence):
+def Enumerate(evidence, location=None):
   """Uses dfVFS to enumerate partitions in a disk / image.
 
   Args:
     evidence: Evidence object to be scanned.
+    location: dfVFS partition location to be scanned
```

Comment:
So we don't need to use this location directly when we call `GetBasePathSpecs()`?  We only need to use it to adjust the options volumes or partitions?

Reply:
The location is passed in through the Volume Scanner Options if specified.